src/operator/crop-inl.h

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file crop-inl.h
+ * \brief
+ * \author Wei Wu
+*/
+#ifndef MXNET_OPERATOR_CROP_INL_H_
+#define MXNET_OPERATOR_CROP_INL_H_
+#include <dmlc/logging.h>
+#include <dmlc/parameter.h>
+#include <mxnet/operator.h>
+#include <cstring>
+#include <map>
+#include <string>
+#include <vector>
+#include <utility>
+#include "./operator_common.h"
+
+namespace mxnet {
+namespace op {
+
+namespace crop_enum {
+enum CropOpInputs {kData, kCropLike};
+enum CropOpOutputs {kOut};
+}  // namespace crop_enum
+
+struct CropParam : public dmlc::Parameter<CropParam> {
+  TShape offset;
+  bool center_crop;
+  DMLC_DECLARE_PARAMETER(CropParam) {
+    int shape[] = {0, 0};
+    DMLC_DECLARE_FIELD(offset).set_default(TShape(shape, shape + 2))
+    .describe("corp offset coordinate: (y, x)");
+    DMLC_DECLARE_FIELD(center_crop).set_default(false)

```

Comment:
No, I mean you could add in the parameters, then we can reuse the op in other applications. Moreover, if we crop the input to the same size, it seems that we don't need it...